MinnPost/object-sync-for-salesforce
# 167 skip fields with no value

## What does this PR do?

When pulling data from Salesforce into WordPress, this skips fields that would be mapped but have no values.

## How do I test this PR?

- create a fieldmap.
- make sure that some of the fields that are not required to create the WordPress object are empty, and that some other not required fields have values.
